Integrations/integration-RsaNetwitnessSecurityAnalytics.yml

```diff
@@ -1083,43 +1083,44 @@ script:
             break;
         case 'nw-list-incidents':
             var incidents = listIncidents(sessionId, args, incidentManagementId);
-            results = { incidents: incidents };
+            results = {Type: entryTypes.note, Contents: { incidents: incidents }, ContentsFormat: formats.json, HumanReadable: tableToMarkdown(command, incidents), ReadableContentsFormat: formats.markdown};
             break;
         case 'nw-get-incident-details':
             var incident = getIncidentById(sessionId, args, incidentManagementId);
-            results = { incident: incident };
+            results = {Type: entryTypes.note, Contents: { incident: incident }, ContentsFormat: formats.json, HumanReadable: tableToMarkdown(command, incident), ReadableContentsFormat: formats.markdown};
             break;
         case 'nw-get-components':
             var components = getComponents(sessionId, args);
-            results = { components: components };
+            results = {Type: entryTypes.note, Contents: { components: components }, ContentsFormat: formats.json, HumanReadable: tableToMarkdown(command, components), ReadableContentsFormat: formats.markdown};
             break;
         case 'nw-get-events':
             var events = getEvents(sessionId, args);
-            results = { events: events };
+            results = {Type: entryTypes.note, Contents: { events: events }, ContentsFormat: formats.json, HumanReadable: tableToMarkdown(command, events), ReadableContentsFormat: formats.markdown};
             break;
         case 'nw-get-event-details':
-            results = getEventDetails(sessionId, args);
+            var eventDetails = getEventDetails(sessionId, args);
+            results = {Type: entryTypes.note, Contents: eventDetails, ContentsFormat: formats.json, HumanReadable: tableToMarkdown(command, eventDetails), ReadableContentsFormat: formats.markdown};
```

Comment:
This is the result when I run this command
![image](https://user-images.githubusercontent.com/7270217/27801975-9f1a2694-6029-11e7-9e7e-dcb05b244e1f.png)